src/languages/de-DE.js

```diff
@@ -350,4 +350,15 @@ export default {
     serwersmsPhoneNumber: "Telefonnummer",
     serwersmsSenderName: "Name des SMS-Absenders (über Kundenportal registriert)",
     "stackfield": "Stackfield",
+    clicksendsms: "ClickSend SMS",
+    apiCredentials: "API Zugangsdaten",
+    smtpDkimSettings: "DKIM Einstellungen",
+    smtpDkimDesc: "Für die Verwendung, schaue bitte in die Nodemailer DKIM {0}.",
```

Comment:
Komma unnötig und ungewöhnlicher Satzbau/Wortwahl => Details zur Konfiguration sind in der Nodemailer DKIM {0} zu finden.